scala3doc/src/dotty/dokka/translators/FilterAttributes.scala

```diff
@@ -43,6 +43,7 @@ object FilterAttributes:
         case Origin.InheritedFrom(name, _) => Map("inherited" -> name)
         case Origin.ImplicitlyAddedBy(name, _) => Map("implicitly" -> s"by $name")
         case Origin.ExtensionFrom(name, _) => Map("extension" -> s"from $name")
+        case Origin.ExportedFrom(name, _) => Map("export" -> s"Exported from $name")
```

Comment:
Where is this used? Why is the previous line `"from $name"`, but this one is `"Exported from $name"`, i.e. why is the "Exported" part necessary?

Reply:
Yeah, I could add some exports to the test cases. 

For this one with `Exported from...` is my fault, will fix

Reply:
Only known export for me at this moment is here 
https://scala3doc.virtuslab.com/pr-10504/self/scala3doc/-a-p-i/dotty.dokka/tasty/-basic-support.html

Reply:
Updated testcases, check out exports here https://scala3doc.virtuslab.com/pr-10504/testcases/-scala3doc%20testcases/tests/exports/-b.html